plugins/telemetry/telemetry.h

```diff
@@ -59,24 +101,36 @@ class Telemetry
         UNKNOWN
     };
 
+    /**
+     * Returns a human readable English string for a flight mode.
+     */
     static const char *flight_mode_str(FlightMode flight_mode);
 
+    /**
+     * Various health flags
+     */
     struct Health {
-        bool gyrometer_calibration_ok;
-        bool accelerometer_calibration_ok;
-        bool magnetometer_calibration_ok;
-        bool level_calibration_ok;
-        bool local_position_ok;
-        bool global_position_ok;
-        bool home_position_ok;
+        bool gyrometer_calibration_ok; /**< true if the gyrometer is calibrated */
+        bool accelerometer_calibration_ok; /**< true if the accelerometer is calibrated */
+        bool magnetometer_calibration_ok; /**< true if the magnetometer is calibrated */
+        bool level_calibration_ok; /**< true if the vehicle has a valid level calibration */
+        bool local_position_ok; /**< true if the local position estimate is good enough */
+        bool global_position_ok; /**< true if the global position estimate is good enough */
+        bool home_position_ok; /**< true if the home position has been initialized properly */
     };
 
+    /**
+     * Remove control status type
+     */
     struct RCStatus {
-        bool available_once;
-        bool lost;
-        float signal_strength_percent;
+        bool available_once; /**< true if an RC signal has been available once */
+        bool lost; /**< true if the RC signal is lost */
+        float signal_strength_percent; /**< signal strength in percent from 0 to 100 */
```

Comment:
As above, I like to separate out range into bracket. Up to you.

Reply:
Done.